### ISSUE TITLE
Keep assets from previous builds by default

### ISSUE THREAD

**Issue by zealot128** [ElMassimo/vite_ruby#177]
Reasoning: during "traditional" capistrano-like deployments the 'public/vite' folder is kept between deployments. It is expected to serve old assets for a "little time" between deployments, or for long running clients, that still request old assets.

The docs write about a separate clobber + clean task, so my expectation was, that Vite-Ruby does not delete assets on normal build. But I found out when trying to deploy a new app with Vite, vite:build etc. will delete previous asset versions every time from the 'public/vite/assets', because the option is set by the plugin:

https://github.com/ElMassimo/vite_ruby/blob/91d2ea44873005a53e97a15e76389feccf99106f/vite-plugin-ruby/src/index.ts#L42

This will lead to missing 404 assets during deployment, and is unexpected, as the "normal" rails asset pipeline + webpacker only add new additional assets and a availability of clean/clobber task implies a non-deleting build IMO.

So my suggestion: 

1. Set emptyOutDir to false when in production (like the line below
2. Or under Deployment in docs tell about this option

<details>
<summary>Workaround:</summary>

```javascript
export default defineConfig({ 
  ...
  build: {
    emptyOutDir: false
  }
```

</details>

<details>
<summary>Versions</summary>

```
    "vite": "^2.7.12",
    "vite-plugin-ruby": "^3.0.7",
```

</details>

**Comment by ElMassimo** [comment]
Hi Stefan!

It makes sense for Vite Ruby to depart from the Vite default, and use `emptyOutDir: false` in production if the user does not explicitly provide a value.

```js
emptyOutDir: userConfig.emptyOutDir ?? !isProd,
```

This should be [documented in the Config section](https://vite-ruby.netlify.app/config/#source-maps-%F0%9F%97%BA).

Pull requests are welcome!

**Comment by ElMassimo** [comment]
Closed by #180, thanks!
